<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrafficMap</class>
 <widget class="QMainWindow" name="TrafficMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="ueGroupWidget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>315</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="ueGroupsLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="UeLabel">
         <property name="text">
          <string>Amount of UE's Groups</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="numberOfUeLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="UeGroups">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>UE Groups: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeUeButton">
         <property name="styleSheet">
          <string notr="true">border-color: #eee;
background-color: #ff4444;</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addUeButton">
         <property name="styleSheet">
          <string notr="true">border-color: #eee;
background-color: #b2d025;</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="rbsWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>551</width>
      <height>401</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>551</width>
      <height>391</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <widget class="QWidget" name="rbsWidget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>551</width>
       <height>391</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>551</width>
       <height>391</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <widget class="QLabel" name="rbsLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>LTE RBS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="srioLabel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>SRIO / ETHERNET</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="ucToolButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>320</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #000;
border-radius:5px;
border-color: #eee;
background-color:#ffd700;</string>
      </property>
      <property name="text">
       <string>UCTool</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>100</width>
        <height>250</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #000;
border-radius:15px;
border-color: #eee;
background-color:#cccccc;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>72</x>
        <y>265</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:17px;
border-color: #eee;
background-color:rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>82</x>
        <y>245</y>
        <width>15</width>
        <height>15</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:6px;
border-color: #eee;
background-color:rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>20</width>
        <height>90</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:9px;
border-color: #eee;
background-color:rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>130</width>
        <height>2</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ffd700</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>130</width>
        <height>2</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ffd700</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>130</width>
        <height>2</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ffd700</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>135</y>
        <width>20</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>L3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>225</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>L2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>325</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>L1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>211</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">:rotate(180, Qt::XAxis);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="pixmap">
       <pixmap resource="../../Resource.qrc">:/Images/MapItems/curved line.svg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>250</y>
        <width>51</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../Resource.qrc">:/Images/MapItems/curved line 2.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="channelModelsButton">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>305</y>
        <width>71</width>
        <height>29</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>10</horstretch>
        <verstretch>10</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <strikeout>false</strikeout>
        <stylestrategy>PreferAntialias</stylestrategy>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #000;
border-radius:5px;
border-color: #eee;
background-color:#ffd700;</string>
      </property>
      <property name="text">
       <string>Channel 
Models</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>0</y>
        <width>251</width>
        <height>391</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #000;
border-radius:5px;
border-color: #eee;
background-color:#cccccc;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>14</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <widget class="QLabel" name="lblLTEsim">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Schoolbook L</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>LTEsim</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ipexButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>260</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000;
border-radius:5px;
border-color: #eee;
background-color:#ffd700;</string>
       </property>
       <property name="text">
        <string>IPEX</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="ubSimButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>340</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000;
border-radius:5px;
border-color: #eee;
background-color:#ffd700;</string>
       </property>
       <property name="text">
        <string>UBsim</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="mmeButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000;
border-radius:5px;
border-color: #eee;
background-color:#666666;</string>
       </property>
       <property name="text">
        <string>MME</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="sgwButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000;
border-radius:5px;
border-color: #eee;
background-color:#666666;</string>
       </property>
       <property name="text">
        <string>SGW</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="paggingButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000;
border-radius:5px;
border-color: #eee;
background-color:#666666;</string>
       </property>
       <property name="text">
        <string>Pagging</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="ueButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>300</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000;
border-radius:5px;
border-color: #eee;
background-color:#ffd700;</string>
       </property>
       <property name="text">
        <string>UE</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="generalSettingsButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000;
border-radius:5px;
border-color: #eee;
background-color:#666666;</string>
       </property>
       <property name="text">
        <string>General settings</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <zorder>label_12</zorder>
     <zorder>label</zorder>
     <zorder>rbsLabel</zorder>
     <zorder>srioLabel</zorder>
     <zorder>ucToolButton</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>line_2</zorder>
     <zorder>line_5</zorder>
     <zorder>label_9</zorder>
     <zorder>label_10</zorder>
     <zorder>label_11</zorder>
     <zorder>label_13</zorder>
     <zorder>channelModelsButton</zorder>
     <zorder>frame</zorder>
     <zorder>line_3</zorder>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>118</width>
      <height>15</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="trafficParamsWidget" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>321</width>
      <height>200</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>150</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>300</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="statisticsButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#statisticsButton { 
	color: #eee;
	border-radius:5px;
	border-color: #eee;
	background-color:#666666;
}
QPushButton#statisticsButton:hover:!pressed{
	color: #000;
	background-color:#ffd700;
}</string>
       </property>
       <property name="text">
        <string>Statistics</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="timeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#timeButton { 
	color: #eee;
	border-radius:5px;
	border-color: #eee;
	background-color:#666666;
}
QPushButton#timeButton:hover:!pressed{
	color: #000;
	background-color:#ffd700;
}</string>
       </property>
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="tunningTrafficButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#tunningTrafficButton { 
	color: #eee;
	border-radius:5px;
	border-color: #eee;
	background-color:#666666;
}
QPushButton#tunningTrafficButton:hover:!pressed{
	color: #000;
	background-color:#ffd700;
}</string>
       </property>
       <property name="text">
        <string>Tuning traffic</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pushButton { 
	color: #eee;
	border-radius:5px;
	border-color: #eee;
	background-color:#666666;
}
QPushButton#pushButton:hover:!pressed{
	color: #000;
	background-color:#ffd700;
}</string>
       </property>
       <property name="text">
        <string>Custom models</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>15</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../Resource.qrc"/>
  <include location="../../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
